frontend/src/components/ShootDetails/GShootKubeconfig.vue

```diff
@@ -45,26 +45,40 @@ SPDX-License-Identifier: Apache-2.0
         </span>
       </div>
     </g-list-item-content>
+    <g-list-item-content v-if="isAdminKubeconfigType">
+      Admin Kubeconfig
+      <div class="text-body-2">
+        <span>
+          Request a kubeconfig valid for {{adminKubeConfigExpiration}}
```

Comment:
I can't remember if you had implemented previously before switching to vue3, but the selected expiration value should be stored in the local storage